backend/cli.py

```diff
@@ -142,13 +138,25 @@ def add_data():
 
 
 @app.command()
-def full_setup(popularity: Optional[float], remove_non_ascii: bool = False):
+@coroutine
+async def flush_cache():
+    await redis.flushdb()
+
+
+@app.command()
+@coroutine
+async def genres_to_cache():
+    await insert_genres_to_cache(get_genres())
+
+
+@app.command()
+@coroutine
+async def full_setup(popularity: Optional[float], remove_non_ascii: bool = False):
```

Comment:
Could the popularity parameter type be written the 'cool' way? :DDDD

Reply:
```py
float | None 🥳 
```

Reply:
Welp nvm, that broke it